### ISSUE TITLE
Can we add this issue to the list? (AMD)

### ISSUE THREAD

**Issue by toomuchvoltage** [danilw/GPU-my-list-of-bugs#1]
This Shadertoy renders wrong on AMD: https://www.shadertoy.com/view/ftKSzy

You have to make a meaningless modification on line 29 (abs() => abs (roundness * 2.0) * 0.5) to make it work properly on AMD. I brought this to their attention over a year ago and it's still like this on the latest Adrenalin drivers.

![GA0QGbOWoAA1HhN](https://github.com/user-attachments/assets/692853c1-6d07-4a26-bb57-5db49f7018fa)

**Comment by danilw** [comment]
Hello.

> renders wrong on AMD

I tested on AMD integrated Vega8 integrated Linux kernel 6.11.8 mesa 24.3.0
No bug for me.

If you want to discuss it - join my discord to communicate in real time to save our time - https://discord.gg/JKyqWgt

> Can we add this issue to the list?

This list is only for my bugs - all bugs here is only what I found - just for reference when I need these links sometime.

I do not add anyone else bugs here, for example FabriceNeyret2 have much larger bug list then me - I not adding his bugs here.

If you want - make your own list with your bugs.
